packages/popmotion-pose/src/dom/pose.ts

```diff
@@ -36,6 +36,8 @@ const createPoseConfig = (
     onPressStart,
     onPressEnd,
     draggable,
+    directionLock,
```

Comment:
would be helpful to add an example how this config option might be used - with description how the behaviour differs when using it or not

if it only makes sense when using in combination with `scrollUntilDragDirection` then it would be good to add a dev warning in case it gets used without it